### DISCUSSION TITLE
`defaultValues: null` cause an Exception

### DISCUSSION THREAD

**Discussion by iron66** [react-hook-form/react-hook-form#2190] [General] [answered]
Not sure is it feature or a bug but:
https://codesandbox.io/s/zealous-fermi-2ol4g?file=/src/index.js

I've set null to state which used as defaultValues in child component. That took pretty long time to understand what's happening

**Comment by bluebill1049** [comment, marked as answer]
we don't support null for the entire defaultValues, empty {} instead.
